app/views/shared/_order_detail_cell.html.haml

```diff
@@ -6,6 +6,10 @@
     - if show_reservation && od.reservation
       %br
       %em= od.reservation
+
+    - if od.product.offline?
```

Comment:
The unfortunate thing is this only reflects current offline status. If I'm looking at the "All" tab of "My Reservations", it might show the warning sign on a reservation completed at a time when the instrument was online, which could be misleading.